daml-lf/api-type-signature/src/main/scala/lf/typesig/PackageSignature.scala

```diff
@@ -32,6 +32,7 @@ final case class PackageMetadata(
   */
 final case class PackageSignature(
     packageId: PackageId,
+    languageVersion: LanguageVersion,
```

Comment:
Naive question, why not a boolean (or a sealed class) flag instead of the language version ?

Reply:
I was going to switch to a boolean but investigated this a bit more and there are some benefits:
- It is more general purpose as it can be inspected for language feature support.
- It seems HTTP-Json needs to be aware of `LanguageVersion` as it is included in `PackageSignature`
- It types better when used in a tuple (as is usually the case).